includes/ShareConfigProvider.php

```
@@ -0,0 +1,110 @@
+			return false;
+		}
+		return preg_match( '/^(?:' . $protocols . ')/i', $url ) === 1;
+	}
+
+	/**
+	 * @param array $arr
+	 * @return bool
+	 */
+	private static function isList( array $arr ): bool {
+		$i = 0;
+		foreach ( $arr as $k => $_ ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$_'. (undefined)

(UnusedLocalVariable)